frame/vamm/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+#[frame_support::pallet]
+pub mod pallet {
+	// ----------------------------------------------------------------------------------------------------
```

Comment:
@KaiserKarel @hussein-aitlahcen are we ok to diverge from what is accepted in terms of such comments in polkadot ecosystem?